src/components/dot-leader/dot-leader.stories.mdx

```diff
@@ -44,5 +44,29 @@ If `c-dot-leader` is an `<a>` element, it will gain some additional styles.
 Although `c-dot-leader` applies to a single row, the pattern works best as part of [a list object](/docs/objects-list--default-story). The dotted lines clearly associate each column even when content lengths vary.
 
 <Canvas>
-  <Story name="List">{listDemo({ topics })}</Story>
+  <Story
+    name="List"
+    parameters={{
+      docs: {
+        source: {
+          code: `{% embed '@cloudfour/objects/list/list.twig' only %}
+  {% block content %}
+    {% for topic in topics %}
+      <li>
+        {% include '@cloudfour/components/dot-leader/dot-leader.twig' with {
+          label: topic.title,
+          count: topic.count,
+          href: topic.url
+        } only %}
+      </li>
+    {% endfor %}
+  {% endblock %}
+{% endembed %}
+`,
```

Comment:
@gerardo-rodriguez Unless I'm mistaken, this looks like it's exactly the contents of the `demo/list.twig` file.

In cases where you want to show the raw template source, it's possible to do so without copying and pasting code.

Near the beginning of the file, you should see the import for the demo itself:

```js
import listDemo from './demo/list.twig';
```

Duplicate this line, but give it a unique name and prepend `!!raw-loader!` to the path:

```js
import listDemo from './demo/list.twig';
import listDemoSource from '!!raw-loader!./demo/list.twig';
```

Then, you can use `listDemoSource` (or whatever you called it) as the value of `docs.source.code`. It should show the contents of the `list.twig` file without compiling them.

More info: https://github.com/webpack-contrib/raw-loader#examples

Reply:
What!? 🤯 

Thanks, @tylersticka, I'll take a look! 🔍 👀

Reply:
Done, thanks, @tylersticka! https://github.com/cloudfour/cloudfour.com-patterns/pull/1356/commits/e38b9e6f1b213f9c8bc9715e482a2231a80a22f2

Reply:
Oh...lint failure... 🔍

Reply:
I was getting a lint error. [I disabled it](https://github.com/cloudfour/cloudfour.com-patterns/pull/1356/commits/cd366c7cf4935fe0b7143c8672d0324f0cd40028). Not sure if disabling this specific use case is frowned upon. Do you have any insight, @tylersticka?

Reply:
Interesting! Here's the [info about the lint rule](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-webpack-loader-syntax.md), and the relevant bit:

> This syntax is non-standard, so it couples the code to Webpack. The recommended way to specify Webpack loader configuration is in a Webpack configuration file.

I get the intent here. `import` is a native feature that works anywhere. By using custom webpack syntax we're breaking that interoperability and tying ourselves to webpack. That said,  it seems unlikely that Storybook would move away from webpack any time soon, and since we're already tied to Storybook it seems like we're already tied to webpack? 

I think the worst case scenario is that a new Storybook version moves away from webpack and then we'd need to either:
- Add our own webpack config.
- Update anywhere we use this syntax to do something else.
- Not update Storybook.

That doesn't seem all that scary? I'd be more concerned about this lint rule being used in our actual components, but it seems fine for a Storybook docs page?

I'm trying to think through alternatives:

1. We could update our webpack config so that you could add a query string to specify a specific loader: (e.g. `import myRawModule from 'myModule?raw'`. I think this could be done using [resource query rules](https://webpack.js.org/configuration/module/#ruleresourcequery)? 
2. Write some custom node code to grab a file's contents and spit them out? That seems like reinventing the wheel for no reason and I'm not sure how it would work with Storybook.

It seems like option 1 above might be slightly cleaner since we're not tying ourselves to webpack as directly? But we'd still be relying on webpack to handle the resource query rules. That solution doesn't seem radically better but it sounds more complicated to get working.

My gut feeling is we should leave this rule enabled globally (since we shouldn't be doing this in our component code) but not worry about disabling it in Storybook doc files.

That said I'm curious if @calebeby or @spaceninja have any other thoughts.

Reply:
Option 1 feels like we're just recreating a native Webpack feature ourselves for no reason, and option 2 does feel like reinventing the wheel for no reason.

> My gut feeling is we should leave this rule enabled globally (since we shouldn't be doing this in our component code) but not worry about disabling it in Storybook doc files.

I agree with this 👍

Reply:
I agree with @Paul-Hebert — leave the rule enabled, and disable it for places where we're intentionally working around it like Storybook docs.

That said, I'd like to see a comment above the eslint-disable line explaining why we're disabling it. Just a sentence or two summarizing what we've said here.

Reply:
Thank you all! I'll add a comment shortly. 👍

Reply:
Comment added! https://github.com/cloudfour/cloudfour.com-patterns/pull/1356/commits/f6178df483e6acc6baf64c21b5bee1fa452ef4b4